app/src/main/java/com/example/marketplace/view/productDetails/ProductDetailsFragment.kt

```diff
@@ -0,0 +1,87 @@
+package com.example.marketplace.view.productDetails
+
```

Comment:
![4](https://user-images.githubusercontent.com/52527199/186385199-03a68a3c-85db-4c3d-9243-cfea3adc4610.png)

Please make sure that the sides of all elements in any screen are aligned

this concept apply to any screen and views